### PR TITLE
[CSP-7569] DDL Docs

### PR DESCRIPTION
**Pull request by Kalininator**
Added DDL/Lookup docs

### CODE REVIEW

File: docs/pages/ddl.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+### Adding a DDL to an input field
+
+To add a DDL to an input field, you need to use the `lookup` property.
+
+```js
+{% raw %}module.exports = {
+  user_id: {
+    type: 'string',
+    description: 'The ID of the user',
```
Comment (EdinaBolkeny) [inline docs/pages/ddl.md:110]: If it would be a connector code review I would say 'It needs full stop at the end of the description'. :)

**Review by EdinaBolkeny — APPROVED** [review]

**Review by jonurry — COMMENTED** [review]
Really useful info 👍🏻 
What do you think about mentioning the utils helpers for generating DDLs and Lookups.
Maybe have a couple of examples of using those?
Can mention the moustached version in utils?
Or, just link to the connector utils repo readme if you don't want to duplicate that info here?

**Comment by Kalininator** [comment]
> Really useful info 👍🏻 What do you think about mentioning the utils helpers for generating DDLs and Lookups. Maybe have a couple of examples of using those? Can mention the moustached version in utils? Or, just link to the connector utils repo readme if you don't want to duplicate that info here?

perhaps in future. atm im trying to document stuff from the perspective of just `falafel` and what the actual falafel requirements are. 

This is part off the SDK initiative really, and will eventually be tying stuff together. Just leaving separate for now

**Review by jonurry — APPROVED** [review]
LGTM

**Pull request merged**